src/Slider.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import styled from 'styled-components'
-import { Range } from 'rc-slider'
```

Comment:
Shouldn't Rollup handle this sort of thing? cc @hakimelek @bertya

Reply:
I like this better than handling it in Rollup for now. However, we should probably lock the version of `rc-slider`.